### ISSUE TITLE
Setup Residual thresholding

### ISSUE THREAD

**Issue by tcollett** [DES-SL/RingFinder2#17]
We need a threshold on the residual images - things with nothing in the residuals shouldn't be shown to citizens.

**Comment by aprajita** [comment]
threshold defined by SNR - right? rather than absolute

**Comment by drphilmarshall** [comment]
Yes, definitely - so we need robust estimators for both image signal and
image noise.
